build.fsx

```diff
@@ -198,24 +196,19 @@ Target "SourceLink" (fun _ ->
 // --------------------------------------------------------------------------------------
 // Build a NuGet package
 
-Target "NuGet" (fun _ ->        
-    packages |> Seq.iter (fun package ->
-    NuGet (fun p -> 
-        { p with   
-            Authors = package.Authors
-            Project = package.Name
-            Summary = package.Summary
-            Description = package.Description
-            Version = buildVersion
-            ReleaseNotes = String.Join(Environment.NewLine, release.Notes)
-            Tags = package.Tags
-            OutputPath = "bin"
-            AccessKey = getBuildParamOrDefault "nugetkey" ""
-            Publish = hasBuildParam "nugetkey"
-            Dependencies = package.Dependencies |> List.map (fun (name,dep) -> (name,dep.Force()))
-        })
-        ("NuGet/" + package.Name + ".nuspec")
-   )
+Target "PaketPack" (fun _ ->
+    Paket.Pack (fun p ->
+      { p with
+          OutputPath = "../../bin/PaketPack"
+          Version = buildVersion
+          ReleaseNotes = toLines release.Notes
+      })
+)
+
+Target "PaketPush" (fun _ ->
+    Paket.Push (fun p -> 
+        { p with 
+            WorkingDir = "../../bin/PaketPack" }) 
```

Comment:
Does this work the same way with regards to the nugetkey? I.e. clearly the push won't work unless it knows my key, but do I still set is as an env variable in my (not checked in) release script?